python/ray/tune/test/trial_runner_test.py

```diff
@@ -1864,6 +1862,29 @@ def count_checkpoints(cdir):
         self.assertEquals(count_checkpoints(tmpdir), 2)
         shutil.rmtree(tmpdir)
 
+    def testIterationCounter(self):
+        def train(config, reporter, steps):
```

Comment:
this doesn't work right? 
 

try running `pytest trial_runner_test.py -k IterationCounter`